cosmicds/components/table/table.vue

```diff
@@ -37,7 +37,7 @@
             <v-icon>{{tool.icon}}</v-icon>
           </v-btn>
         </template>
-        {{tool.tooltip}}
+        <p style="max-width: 40ch;" v-html="tool.tooltip">  </p>
```

Comment:
It doesn't affect the functionality, but I think we should aim to avoid inline CSS unless it really offers an advantage. Something like `class="tooltip"` and `.tooltip { max-width: 40ch }` in the `style` tag seems like a better approach to me.